tests/testthat/test_zzz.R

```diff
@@ -9,3 +9,16 @@ test_that("test get_i18n_config", {
   expect_type(get_i18n_config("cultural_bignumer_mark"), "character")
   expect_error(get_i18n_config("cultural_bignumer_markx"))
 })
+
+test_that("Test .onLoad", {
+  # Assert
+  expect_equal(
+    class(.i18_config),
+    c("list")
+  )
+  expect_equal(length(.i18_config), 3)
+  expect_equal(
+    names(.i18_config),
+    c("cultural_date_format", "cultural_bignumer_mark", "cultural_punctuation_mark")
```

Comment:
Since the function takes values from [this config file](https://github.com/Appsilon/shiny.i18n/blob/main/inst/config.yaml), you can also check if they are passed correctly.

Reply:
Updated